samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.h

```diff
@@ -23,13 +24,16 @@ struct generic_onoff_state {
 
 struct generic_level_state {
 	int level;
+	s32_t current_delta;
+	s32_t last_delta;
 	u8_t model_instance;
 	u8_t last_tid;
 	u16_t last_tx_addr;
 };
 
 struct generic_onpowerup_state {
 	u8_t onpowerup;
+	u8_t tmp;
```

Comment:
ditto

Reply:
Yes. 
Kindly consider same explanation as above.

Since have to access it in **gen_onoff_set_unack()** as well as in **gen_onoff_set()**